index.html

```diff
@@ -218,11 +218,17 @@ <h3>{{RTCRtpSender/setParameters()}}</h3>
            conditions (step 4 within step 6) for an {{InvalidModificationError}}:
```

Comment:
This confused me when I first read it, because context is missing.
Suggest "Add the following to the conditions under which the operation causes a promise rejected with an InvalidModificationError".

Reply:
Fixed.